Running _two_thousand.py

```diff
@@ -0,0 +1,19 @@
+import time
+
+
+def timer(f, *args, **kwargs):
```

Comment:
Don't use abbreviations in variable names -
https://softwareengineering.stackexchange.com/questions/72593/how-to-abbreviate-variable-names